src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -327,6 +362,18 @@ const analyticsEvents = {
       action: 'blur',
       event: 'int-memorable-date-blur',
       prefix: 'memorable-date',
+      ga4: {
+        event: 'interaction',
+        component_name: 'va-memorable-date',
+        custom_string_1: 'component-library',
+        mapping: {
+          'memorable-date-label': 'heading_1',
+          'memorable-date-year': 'value',
+          'memorable-date-month': 'custom_number_1',
+          'memorable-date-day': 'custom_number_2',
```

Comment:
Similar questions as above. Label doesn't seem like a heading to me. Custom numers for month and day but not year?

```
'memorable-date-label': 'heading_1',
'memorable-date-year': 'value',
'memorable-date-month': 'custom_number_1',
'memorable-date-day': 'custom_number_2',
```